api/provider/aws/capacity.go

```diff
@@ -34,7 +34,10 @@ func (p *AWSProvider) CapacityGet() (*structs.Capacity, error) {
 			if *resource.Name == "MEMORY" {
 				capacity.InstanceMemory = *resource.IntegerValue
 				capacity.ClusterMemory += *resource.IntegerValue
-				break
+			}
+			if *resource.Name == "CPU" {
```

Comment:
I took this calculation out recently, and added pulling instance CPU metrics from CloudWatch. 

I think this change back could be good though... There are 2 CPU concepts. How much is reserved due to scheduling and how much is utilized according to the kernel.

I believe this is similar to the `Memory` vs `Size` values in the API.